app/spec/services/github_checks_verifier_spec.rb

```diff
@@ -105,4 +108,82 @@
       expect(output).to include("check_completed: completed (success)")
     end
   end
+
+  describe "#apply_filters" do
+    it "filters out all but check_name" do
+      checks = [
+        OpenStruct.new(name: "check_name", status: "queued"),
```

Comment:
What's the advantage of using ostruct in this file?

Reply:
It's only to reproduce the behavior from what we get from the API response (we get Objects that respond to those methods instead of hashes). But we could easily make them doubles. Probably using doubles would be a lighter solution

Reply:
up to you!

Reply:
Actually, RSpec doubles are not the lighter solution. Take a look to [this benchmark](https://gist.github.com/jgaskins/3892455) 
I'd stay with OpenStruct